src/dtflw/storage/fs.py

```
@@ -1,12 +1,11 @@
 from __future__ import annotations
 from pyspark.sql.dataframe import DataFrame
 from pyspark.sql.session import SparkSession
-from dtflw.databricks import get_dbutils
 import fnmatch
 from abc import ABC, abstractmethod
 
 
-def file_exists(abs_path: str, dbutils=None) -> bool:
+def file_exists(abs_path: str, dbutils) -> bool:
```

if dbutils is mandatory now, we need to raise an exception in case it's None